executor/api/rest/utils.go

```diff
@@ -14,3 +24,83 @@ func ExtractRouteAsJsonArray(msg []byte) ([]int, error) {
 		return nil, err
 	}
 }
+
+func embedSeldonDeploymentValuesInJson(namespace string, sdepName string, openapiInterface *interface{}) error {
```

Comment:
Would this be simpler using Go templates (https://golang.org/pkg/text/template/)?

Reply:
I had a look at templates! Especially if I could handle them with the mux `render` method, the only challenge is that it does require removing json fields, which becomes quite fiddly in templates. Ideally we can move towards potentially teh mux + render method, and I have seen we can simplify the OpenAPI schema, but that probably has to be done on a broader work updating the current openAPI

Reply:
Got it. Using `mux` and `render` makes sense, but I agree that it's probably out of the scope of this PR.